cwlavro-tools/src/main/java/io/cwl/avro/CWL.java

```diff
@@ -145,42 +146,114 @@ public static Object getStub(final Object type, final String value) {
     /**
      * @return a gson instance that can properly convert CWL tools into a typesafe Java object
      */
-    public static Gson getTypeSafeCWLToolDocument() {
-        final Type hintType = new TypeToken<List<Any>>() {}.getType();
+    public static Gson getTypeSafeCWLToolDocument() throws GsonBuildException {
+        final java.lang.reflect.Type hintType = new TypeToken<List<Any>>() {}.getType();
```

Comment:
Is there a reason this switches from Type to java.lang.reflect.Type without the import?